packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/configurations/ModelConfiguration.java

```diff
@@ -28,6 +29,9 @@ public class ModelConfiguration extends TerariumAsset {
 	@TSOptional
 	private UUID simulationId;
 
+	@TSOptional
+	private Date temporalContextDate;
```

Comment:
Note java.util.date has a lot of things deprecated and considered somewhat obsolete ... what are you planning to do with it?

Reply:
Im storing the date the user wants to set timestep 0 at on the ModelConfiguration.  If they want April 10th, 2024 at timestep 0 they select that and it is stored on the model configuration.  Maybe I should use java.sql.Timestamp instead? We seem to use that a lot

Reply:
yeah I think probably "java.sql.Timestamp"

Reply:
For what its worth `java.sql.Timestamp` is just a convenience wrapper around `java.util.date` and a few versions back Java basically incorperated JodaTime's library into a new `java.time` package.... but thats just a fun fact for our next pub trivia ;)

`Timestamp` is used everywhere else, yeah may as well use it